cmd/gaiad/cmd/root.go

```diff
@@ -77,51 +90,82 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 	return rootCmd, encodingConfig
 }
 
+// initTendermintConfig helps to override default Tendermint Config values.
+// return tmcfg.DefaultConfig if no custom configuration is required for the application.
+func initTendermintConfig() *tmcfg.Config {
+	cfg := tmcfg.DefaultConfig()
+
+	// these values put a higher strain on node memory
```

Comment:
I think you want these commented out lines to stay, right?
in that case, I'd suggest making it more clear under what circumstances one wants to uncomment them

Reply:
I think this file is a boilerplate from the `simapp` config in the SDK.

This comment appears IBC, ICS and SDK without more context. I guess it's just a warning in case you change these configs, maybe to improve somehow your node performances during consensus(?)